DrX/Sources/Core/ObservableType+Relay.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ObservableType+Relay.swift
+//  DrX
+//
+//  Created by Kevin Li on 2/6/20.
+//  Copyright © 2020 Duet Health. All rights reserved.
+//
+
+import Foundation
+import RxRelay
+import RxSwift
+
+extension ObservableType {
+    func bind<R: RelayType>(to relay: R) -> Disposable where R.Element == Element {
+        bind(to: relay.asObserver())
+    }
+    
+    func bind<R: RelayType>(to relay: R) -> Disposable where R.Element == Element? {
+        map(Optional.some)
+            .bind(to: relay.asObserver())
+    }
```

Comment:
These generic functions make the AnyRelay file unnecessary—try out removing those definitions and making sure they still work.

Reply:
holy shit the thing that threw me off was u using "to take" instead of "that take"